irods/connection.py

```diff
@@ -59,6 +59,9 @@ def __init__(self, pool, account):
             raise ValueError("Unknown authentication scheme %s" % scheme)
         self.last_used_time = datetime.datetime.now()
```

Comment:
no need to set it twice

Reply:
In the constructor, I am setting both create_time and last_used_time, which initially is equal to create_time

Reply:
Just realized what you meant. Fixed it.